polaris_wyre/helpers/exceptions.py

```diff
@@ -0,0 +1,5 @@
+from requests.exceptions import HTTPError
+
+
+class WyreAPIError(HTTPError):
+    pass
```

Comment:
Wouldn't be useful to set a default status_code or something like that for the exception? (If I'm correct and this `HTTPError` is the kind of exception that has this field)

Reply:
Yes! You can see the use of this class [here](https://github.com/CheesecakeLabs/django-polaris-wyre/pull/4/files/34aa895b6bf62bebab6b337d05a33a12106c67cc#diff-2747faa85e367a7a9f719cc992b7925a199a06e563e028db1d2f0b877ac6c4c5R40-R43). We show the status code, reason, and response text. What do you think?

Reply:
Oh, so you store the entire response in this Exception?

Reply:
Yes. This class has the same behavior as the HTTPError class. But, the exception raised is easier to understand and know where it happened.